src/Library/demos/Spinner/main.js

```diff
@@ -0,0 +1,13 @@
+const start = workbench.builder.get_object("start");
+const stop = workbench.builder.get_object("stop");
+const spinner = workbench.builder.get_object("spinner");
+
+start.connect("clicked", () => {
+  spinner.start();
+  spinner.set_spinning(true);
```

Comment:
they do the the same thing

also please use properties when possible

```suggestion
  spinner.spinning = false;
```